hack/release.sh

```diff
@@ -267,7 +267,7 @@ EOF
 
 	# Add the DEB package to the APT repo
 	DEBFILE=bundles/$VERSION/ubuntu/lxc-docker*.deb
```

Comment:
Quotes are intentionally not on this line specifically so that the glob expands. :confused: